tests/sentry/integrations/slack/test_image_builder.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+
+from sentry.integrations.slack.message_builder.image_block_builder import ImageBlockBuilder
+from sentry.issues.grouptype import PerformanceP95EndpointRegressionGroupType
+from sentry.models.group import Group
+from sentry.testutils.cases import MetricsEnhancedPerformanceTestCase
+from sentry.testutils.helpers.datetime import before_now
+from sentry.testutils.helpers.features import with_feature
+from tests.sentry.issues.test_utils import OccurrenceTestMixin
+
+
+class TestSlackImageBlockBuilder(MetricsEnhancedPerformanceTestCase, OccurrenceTestMixin):
+    def setUp(self):
+        super().setUp()
+        self.features = {
+            "organizations:performance-use-metrics": True,
+            "organizations:slack-endpoint-regression-image": True,
+        }
+
+    @with_feature("organizations:slack-endpoint-regression-image")
+    def test_image_block(self):
+        for i in range(10):
```

Comment:
not sure if this is the right way to create the transactions but with this the endpoint doesn't return "p95(transaction.duration)". Here's the error I get:

```
ERROR    sentry.integrations.slack:image_block_builder.py:76 build_endpoint_regression_image.failed
Traceback (most recent call last):
  File "/Users/athena1/code/sentry/src/sentry/integrations/slack/message_builder/image_block_builder.py", line 65, in _build_endpoint_regression_image_block
    "percentileData": resp.data["p95(transaction.duration)"],
                      ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'p95(transaction.duration)'
```

Because response has empty data:
```
response {'data': [], 'isMetricsData': True, 'start': 1713978900, 'end': 1713978900, 'meta': {'fields': {'time': 'date', 'count': 'integer'}, 'units': {'time': None, 'count': None}, 'isMetricsData': True, 'isMetricsExtractedData': False, 'tips': {}, 'datasetReason': 'unchanged', 'dataset': 'metrics'}}
```